backend/app/services/execution/polling_dispatcher.py

```
@@ -144,28 +137,20 @@ async def dispatch_polling(
             try:
                 status_result = await gemini_client.get_interaction_status(job_id)
             except GeminiInteractionError:
-                logger.warning(
-                    f"[PollingDispatcher] Status check failed, retrying..."
-                )
+                logger.warning(f"[PollingDispatcher] Status check failed, retrying...")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)